File: Go/Recursions/fibonacciSeries.go
package main

import "fmt"

func fibonacciSeries(n int) {
	var a int = 0
	var b int = 1

	for a < n {
		fmt.Printf("%d ", a)

		temp := a
		a = b
		b = temp + b
	}
	fmt.Println()
}

File: JavaScript/Math/fibonacciSeries.js
const fibonacciSeries = (n) => {
    let a = 0;
    let b = 1;
    let result = [];

    while(a < n) {
        result.push(a);
        [a, b] = [b, a + b];
    }
    return result;
}


console.log(fibonacciSeries(100));